It has no effect when used on the server and reports an error

It has no effect when used on the server and reports an error
```
[03:32:28 ERROR]: Error executing task on Server
java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:156) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1321) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:214) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:130) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1303) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1297) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:140) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1259) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1264) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1135) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:346) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.common.block.entity.CustomCuttingBoardBlockEntity.processStoredStackUsingTool(CustomCuttingBoardBlockEntity.java:46) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.server.ServerPayloadHandler.handleCut(ServerPayloadHandler.java:20) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.ServerPayloadContext.enqueueWork(ServerPayloadContext.java:32) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.handle(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:60) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        ... 14 more
Caused by: java.lang.ClassNotFoundException: org.apache.commons.compress.utils.Lists
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:250) ~[securejarhandler-3.0.8-m.jar:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.common.block.entity.CustomCuttingBoardBlockEntity.processStoredStackUsingTool(CustomCuttingBoardBlockEntity.java:46) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.server.ServerPayloadHandler.handleCut(ServerPayloadHandler.java:20) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.ServerPayloadContext.enqueueWork(ServerPayloadContext.java:32) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.handle(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:60) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        ... 14 more
[03:32:29 ERROR]: Failed to process a synchronized task of the payload: cuttingdelight:cut
java.util.concurrent.CompletionException: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315) ~[?:?]
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320) ~[?:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1770) ~[?:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:156) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:23) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1321) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:214) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:130) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1303) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1297) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:140) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.managedBlock(MinecraftServer.java:1259) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1264) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1135) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.server.MinecraftServer.lambda$spin$2(MinecraftServer.java:346) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/compress/utils/Lists
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.common.block.entity.CustomCuttingBoardBlockEntity.processStoredStackUsingTool(CustomCuttingBoardBlockEntity.java:46) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.server.ServerPayloadHandler.handleCut(ServerPayloadHandler.java:20) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.ServerPayloadContext.enqueueWork(ServerPayloadContext.java:32) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.handle(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:60) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        ... 14 more
Caused by: java.lang.ClassNotFoundException: org.apache.commons.compress.utils.Lists
        at cpw.mods.securejarhandler/cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:250) ~[securejarhandler-3.0.8-m.jar:?]
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.common.block.entity.CustomCuttingBoardBlockEntity.processStoredStackUsingTool(CustomCuttingBoardBlockEntity.java:46) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/cuttingdelight@1.0.0-beta/dev.jaronline.cuttingdelight.server.ServerPayloadHandler.handleCut(ServerPayloadHandler.java:20) ~[%5BD%5D%5B农业%5D%5B农夫乐事%5D%5B高级砧板%5Dcuttingdelight-1.0.0-beta.jar%23459!/:?]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.ServerPayloadContext.enqueueWork(ServerPayloadContext.java:32) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.handle(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/neoforge@21.1.213/net.neoforged.neoforge.network.handling.MainThreadPayloadHandler.lambda$handle$0(MainThreadPayloadHandler.java:16) ~[neoforge-21.1.213-universal.jar%23391!/:1.21.1]
        at TRANSFORMER/minecraft@1.21.1/net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:60) ~[server-1.21.1-20240808.144430-srg.jar%23390!/:?]
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        ... 14 more

```

--- a/src/main/java/dev/jaronline/cuttingdelight/common/block/entity/CustomCuttingBoardBlockEntity.java
+++ b/src/main/java/dev/jaronline/cuttingdelight/common/block/entity/CustomCuttingBoardBlockEntity.java
@@ -14,7 +14,6 @@ import net.minecraft.world.item.enchantment.EnchantmentHelper;
 import net.minecraft.world.item.enchantment.Enchantments;
 import net.minecraft.world.level.block.entity.BlockEntityType;
 import net.minecraft.world.level.block.state.BlockState;
-import org.apache.commons.compress.utils.Lists;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 import vectorwing.farmersdelight.common.block.entity.CuttingBoardBlockEntity;
@@ -22,6 +21,7 @@ import vectorwing.farmersdelight.common.crafting.CuttingBoardRecipe;
 import vectorwing.farmersdelight.common.registry.ModAdvancements;
 import vectorwing.farmersdelight.common.utility.ItemUtils;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class CustomCuttingBoardBlockEntity extends CuttingBoardBlockEntity {
@@ -43,7 +43,7 @@ public class CustomCuttingBoardBlockEntity extends CuttingBoardBlockEntity {
             return false;
         }
 
-        List<ItemStack> results = Lists.newArrayList();
+        List<ItemStack> results = new ArrayList<>();
         int fortuneLevel = EnchantmentHelper.getTagEnchantmentLevel(level.holder(Enchantments.FORTUNE).get(), toolStack);
         for (int i = 0; i < itemCount; i++) {
             results.addAll(recipe.rollResults(level.random, fortuneLevel));

